templates/c/.github/workflows/release.hippo.yml

```diff
@@ -42,22 +42,22 @@ jobs:
     - name: Build release
       run: make WASI_SDK=wasi-sdk-${{ matrix.wasi_sdk_version.major }}.${{ matrix.wasi_sdk_version.minor }}
 
-    - name: Install hippofactory
+    - name: Install hippo
```

Comment:
This has the name `Install Hippo CLI` in the file above - I don't think it matters which but might be good to keep them the same across all four files.